tests/test_s3.py

```diff
@@ -104,6 +107,94 @@ def _create_s3_uploader(
         return uploader_class(integration, client_class=client_class)
 
 
+class S3UploaderIntegrationTest(S3UploaderTest):
+    MINIO_PORT = 9000
+    MINIO_ENDPOINT_URL = 'http://localhost:{0}'.format(MINIO_PORT)
+    MINIO_USER = 'minioadmin'
+    MINIO_PASSWORD = 'minioadmin'
```

Comment:
I don't think these should be hard-coded, since they might conflict with a developers environment and would require tracking down and changing code to get tests running. You could make it more visible and with an environment variable that's documented alongside `SIMPLIFIED_TEST_DATABASE`, which is used in a similar manner.

Reply:
I replaced hard-coded credentials with environment variables